service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/rest/JsonRestEndpoint.scala

```diff
@@ -126,7 +126,7 @@ class JsonRestEndpoint(
                                 val params: Seq[AnyRef] = m
                                   .getParameters
                                   .headOption
-                                  .map { _ => request.getParts }
+                                  .map { _ => request.getParts.asScala }
                                   .toSeq
                                 m.invoke(o, params: _*) match {
```

Comment:
could be just `m.invoke(o, request.getParts.asScala)`

Reply:
done.